pkg/reconciler/ingress/ingress.go

```diff
@@ -270,10 +270,9 @@ func (r *Reconciler) reconcileVirtualServices(ctx context.Context, ing *v1alpha1
 	}
 
 	// Now, remove the extra ones.
-	// TODO(https://github.com/knative/serving/issues/6363):  Switch to use networking.IngressLabelKey instead.
 	vses, err := r.virtualServiceLister.VirtualServices(resources.VirtualServiceNamespace(ing)).List(
```

Comment:
We can remove the `VirtualServiceNamespace` and just use `ing.Namespace` now I think. It's used in very few places and would make this piece of code more clear, especially as it makes clear that we can drop the namespaceLabelKey below, I think?

Reply:
I have cleaned up some referenced in ingress.go and as well in tests.